src/randgraphs.jl

```diff
@@ -193,6 +193,130 @@ function barabasi_albert(n::Integer, k::Integer; is_directed=false, seed::Int=-1
     return g
 end
 
+"""
+    static_fitness_model{T<:Real}(m::Int, fitness::Vector{T}; seed::Int=-1)
+
+Generates a non-growing random undirected graph with `length(fitness)` nodes and `m` edges,
+in which the edge probabilities proportional to node fitness scores.
```

Comment:
Could you change this sentence to

Generates a random undirected graph with `length(fitness)` nodes and `m` edges,
in which the probability of the existence of edge `(i, j)` is proportional to `fitness[i]*fitness[j]`.